src/com/sap/piper/tools/neo/NeoCommandHelper.groovy

```diff
@@ -87,6 +90,15 @@ class NeoCommandHelper {
         return "--source ${BashUtils.quoteAndEscape(source)}"
     }
 
+    private String extensions() {
+        if(! this.extensions) return ''
+        def quotedExtensions = []
+        for(def ext : this.extensions) {
```

Comment:
could be inlined using `.collect`: 
```
this.extensions.collect({it -> "'${it}'"}).join(',')
```

Reply:
done. I had it similar with `stream()` somehow in-between. This worked on the level of plain junit, but not on Jenkins. On Jenkins this ended up in a strange stack trace. Hence I used that approach here. But your approach works fine, this is much more straight-forward. Thanks.